changes/6439-gamaleonardo.md

```diff
@@ -0,0 +1 @@
+Adding `enum` validation error to the documentation. It was missing there. 
```

Comment:
We don't need a change file for this kind of change. please remove it.

Reply:
Ok, thank you for that